exo-sync/bin/setup

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-npm install --loglevel error --depth 0
+yarn install
```

Comment:
@kevgo, i think we should leave installing `yarn` up to the user since its installation is [different for each platform](https://yarnpkg.com/en/docs/install#mac-tab).